pyt/lattice.py

```diff
@@ -6,35 +6,34 @@ def __init__(self, cfg_nodes, analysis_type):
         self.el2bv = dict()  # Element to bitvector dictionary
         self.bv2el = list()  # Bitvector to element list
         for i, e in enumerate(analysis_type.get_lattice_elements(cfg_nodes)):
+            # Give each element a unique shift of 1
             self.el2bv[e] = 0b1 << i
-            self.bv2el.append(e)
-        self.bv2el = list(reversed(self.bv2el))
+            self.bv2el.insert(0, e)
 
     def get_elements(self, number):
-        r = list()
-
         if number == 0:
-            return r
+            return []
```

Comment:
The other places in the codebase where we use `[]` are when we pass an empty list as an argument (in base_cfg.py and flask_adaptor.py) and one place where we do `while q != []:` in fixed_point.py. I kind of like `[]` when comparing, passing them as args and returning them as values. I completely agree that `list()` looks nicer during an assignment. Do you only disagree in the case of returning them as values? I don't have a strong opinion on this, but it's good to be on the same page as I wrote more code :)